cms/settings/dev.py

```diff
@@ -24,6 +25,14 @@
     }
 }
 
+# we turn off basic auth when testing because it breaks the test suite
+if not os.environ.get("NO_BASIC_AUTH"):
+    MIDDLEWARE += ["baipw.middleware.BasicAuthIPWhitelistMiddleware"]
+BASIC_AUTH_LOGIN = os.environ.get("BASIC_AUTH_LOGIN", "nhsx")
+BASIC_AUTH_PASSWORD = os.environ.get(
+    "BASIC_AUTH_PASSWORD", "hardcodedpasswordpleasechange"
```

Comment:
I'd be tempted to leave no default here, so if one isn't set it just fails (and then we'll get poked via Rollbar etc)

Reply:
I agree, but I'm going to leave this one for later; this currently blocks sorting out staging and I'd like to get that sorted today.